apps/nextra/pages/en/build/smart-contracts/book/variables.mdx

```diff
@@ -619,6 +619,48 @@ script {
 
 For more details, see [Move references](references.mdx).
 
+### Compound Assignments
+
+_Since language version 2.1_
+
+Move also supports compound assignment operators.
```

Comment:
add:

These are like an assignment to a variable, or a mutation through a reference, except that the assigned location must already have a value, which is read and operated on before being stored back into the location.

Currently these are only applicable to numeric values.

Reply:
done